comp/core/healthprobe/healthprobeimpl/healthprobe_test.go

```diff
@@ -30,20 +30,35 @@ func TestServer(t *testing.T) {
 		"health_port": port,
 	}
 
-	_, err := newHealthProbe(lc,
+	probe, err := newHealthProbe(lc,
 		fxutil.Test[dependencies](
 			t,
 			logimpl.MockModule(),
 			config.MockModule(),
 			fx.Replace(config.MockParams{Overrides: overrides}),
-		))
+		),
+	)
+	assert.Nil(t, err)
```

Comment:
It is usually better to use `assert.NoError` because when `err != nil` there error message is better.